### DISCUSSION TITLE
Password Strength Bar below password field

### DISCUSSION THREAD

**Discussion by ziming** [filamentphp/filament#3370] [Ideas]
It will be nice if the password field can have a password strength bar underneath, as for how the strength is to be calculated, it can be a closure we pass in to determine, be it our own mechanism or by hooking it to a zxcvbn package

**Comment by danharrin** [comment]
We don't have a password field, this would be a good plugin but not something we'd add to core

You could PR this existing password field with other functionalities - https://filamentphp.com/plugins/password-reveal-input
